spec/requests/api/v1/products_spec.rb

```diff
@@ -36,6 +38,7 @@
       tags 'Products'
       consumes 'application/json'
       produces 'application/json'
+      security [Bearer: []]
```

Comment:
It's an open method for all users and crawled bots. We don't need authorization here

Reply:
the same